### ISSUE TITLE
`shh.haveIdentity()` does only accept  decimal version of identities

### ISSUE THREAD

**Issue by frozeman** [ethereum/ethereum.js#40]
A new identity can only be verified using toDecimal e.g.:

```js
var myid = web3.shh.newIdentity()

web3.shh.haveIdentity(web3.toDecimal(myid));
```

Which returns `TRUE`

**Comment by frozeman** [comment]
i'm alos not so sure about the identity transformation. Why i have to transform it to be passed to `haveIdentity()` but not when i add it to the `from` or topic field?

IMHO this should be consistent.

btw i find `hasIdentity()` a better name :)

**Comment by debris** [comment]
fixed by gavin in [cpp-ethereum 5b5485b4c4ba563720044fc1c21f6d9740800ef3](https://github.com/ethereum/cpp-ethereum/commit/5b5485b4c4ba563720044fc1c21f6d9740800ef3)

now, you can use it like this:
```javascript
var identity = web3.shh.newIdentity();
console.log(identity); // "0xb0ebfc9dbf9fab9ed8dc61890b2acd09690bb3f1753b9352f19455fd6d23bcc0e6498c2d45d69ee48de90a641afb00105ba1895d532e0d85d333e79612cde2bf"

var haveIdentity = web3.shh.haveIdentity(identity);
console.log(haveIdentity); // true
```
